charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -91,6 +91,9 @@ spec:
               etcd_host:
                 type: string
                 default: ""
+              etcd_host_version:
+                type: string
+                default: "v2"
```

Comment:
@Lucasgranet please pay attention that's able to set any Patroni environment variable via `Env` section of the `Postgresql` spec now.